cli/tox.ini

```diff
@@ -155,7 +155,6 @@ commands =
     pylint \
     setup.py \
     src/pcluster \
-    ../util/ \
```

Comment:
why remove this?

Reply:
We are not really testing util folder with pytest, because there is no __init__.py file inside. 
I tried to add the file but there are too many errors/warnings to be fixed that cannot be part of this patch.